committee-steering/governance/sig-governance.md

```diff
@@ -90,28 +90,36 @@ Subproject contributors (as applicable).
 - *SHOULD* identify, track, and maintain the SIGs enhancements for current
   release and serve as point of contact for the release team, but *MAY* delegate
    to another Lead to fulfill these responsibilities
-  - *MAY* delegate the creation of a SIG roadmap to other Leads
-  - *MUST* organize a main group meeting and make sure [sigs.yaml] is up to date
-  including subprojects and their meeting information but *SHOULD* delegate the
-  need for subproject meetings to subproject owners  
-  - *SHOULD* facilitate meetings but *MAY* delegate to other Leads or future
-  chairs/chairs in training
+- *MUST* ensure the SIG roadmap is shared regularly. *MAY* delegate the creation
+  of a SIG roadmap to other Leads
+- *MUST* watch for the active issues and PRs trends
+  and *SHOULD* coordinate triage, backlog grooming, assignments, and status checks
+- *MUST* organize a main group meeting and make sure [sigs.yaml] is up to date
+including subprojects and their meeting information but *SHOULD* delegate the
+need for subproject meetings to subproject owners  
+- *SHOULD* facilitate meetings but *MAY* delegate to other Leads or future
+chairs/chairs in training
+- *MUST* ensure the good contributors experience, *MAY* delegate those tasks
+  to contributors:
   - *MUST* ensure there is a maintained CONTRIBUTING.md document in the
-  appropriate SIG folder if the contributor experience or on-boarding knowledge
-  is different than in the general [contributor guide]. *MAY* delegate to
-  contributors to create or update.
-  - *MUST* organize KubeCon/CloudNativeCon Intros and Deep Dives with CNCF Event
-   staff and approve presented content but *MAY* delegate to other contributors
-   to create material and present  
-  - *MUST* ensure meetings are recorded and made available
-  - *MUST* report activity with the community via k-dev mailing list at least
-  once a quarter (slides, video from kubecon, etc)
-  - *MUST* coordinate sponsored working group updates to the SIG and the wider
+    appropriate SIG folder if the contributor experience or on-boarding
+    knowledge is different than in the general [contributor guide]. 
+  - *MUST* curate "help wanted" and "good first issue" issues to simplify new
```

Comment:
I don't get why this is a MUST but updating the SIG's docs is only a SHOULD?

Reply:
updating docs can handled by SIG Docs or other maintainers. Chair may be a point of contact for SIG Docs as listed in other requirement. I don't think curating docs is a chair's primary concern. Helping new contributors must be a concern. 

*For context, I also thought for some reasons that the number of "help wanted" issues was an the annual report questions. But checked now and I was wrong.*

Do you think this should be *SHOULD* or docs should be *MUST*?

Reply:
Again, this seems like something that members of the SIG should be collectively doing. I'm not sure the chairs are a good fit for this in all SIGs. TLs and Area Leads / OWNERS are more likely to understand if issues are appropriate across subprojects.

Reply:
As-is, the tools for this make clear we have high expectations for labeling these, that require local context and approval of outlined approach, which seem best left to subproject OWNERS.

Reply:
should chair be responsible for delegating/tracking those? There are conversations on how to promote contributors thru the contributors ladder. Help wanted is a beginning of this ladder.

If this is TL responsibility, I'm happy to write it as such.